packages/ilib-loctool-webos-javascript/test/JavaScriptFile.test.js

```diff
@@ -362,8 +362,7 @@ describe("javascriptfile", function() {
             type: jsft
         });
         expect(j).toBeTruthy();
-
-        j.parse('\tRB.getString("This is a test"); // i18n: this is a translator\'s comment\n\tfoo("This is not");');
+        j.parse('\tRB.getString("This is a test"); // i18n this is a translator\'s comment\n\tfoo("This is not");');
```

Comment:
Add a few unit tests to test with AND without the colon after the i18n part

Reply:
yep. I added more test cases.